addons/acr/CfgVehicles.hpp

```diff
@@ -0,0 +1,187 @@
+        respawnWeapons[] = {};
+        magazines[] = {};
+        respawnMagazines[] = {};
+        items[] = {};
+        respawnItems[] = {};
+        linkedItems[] = {};
+        respawnLinkedItems[] = {};
+        accuracy = 2.3;
+        camouflage = 1.2;
+        cost = 90000;
+        threat[] = {1,0.1,0.1};
+        class Wounds {
+            tex[] = {};
+            mat[] = {
+                "A3\Characters_F\BLUFOR\Data\clothing1.rvmat",
+                "A3\Characters_F\BLUFOR\Data\clothing1_injury.rvmat",
+                "A3\Characters_F\BLUFOR\Data\clothing1_injury.rvmat",
+                "A3\Characters_F\Common\Data\basicbody.rvmat",
+                "A3\Characters_F\Common\Data\basicbody_injury.rvmat",
+                "A3\Characters_F\Common\Data\basicbody_injury.rvmat",
+                "A3\Characters_F\Common\Data\coveralls.rvmat",
+                "A3\Characters_F\Common\Data\coveralls_injury.rvmat",
+                "A3\Characters_F\Common\Data\coveralls_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_bald_muscular.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_bald_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_bald_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_black_bald_muscular.rvmat",
+                "A3\Characters_F\Heads\Data\hl_black_bald_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_black_bald_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_hairy_muscular.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_hairy_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_hairy_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_old.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_old_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_old_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_asian_bald_muscular.rvmat",
+                "A3\Characters_F\Heads\Data\hl_asian_bald_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_asian_bald_muscular_injury.rvmat"
+            };
+        };
+    };
+
+    #include "units\CfgUnits.hpp"
+    #include "vehicles\CfgAntiAir.hpp"
+    #include "vehicles\CfgArtillery.hpp"
+    #include "vehicles\CfgBuran.hpp"
+    #include "vehicles\CfgBVP.hpp"
+    #include "vehicles\CfgDingo.hpp"
+    #include "vehicles\CfgDrone.hpp"
+    #include "vehicles\CfgHeli.hpp"
+    #include "vehicles\CfgJet.hpp"
+    #include "vehicles\CfgPandur.hpp"
+    #include "vehicles\CfgT72.hpp"
+    #include "vehicles\CfgStatic.hpp"
+    #include "vehicles\CfgTatra.hpp"
+
+    class CUP_B_Kombat_Radio_Olive;
+    class UKSF_ACR_Squad_Leader_pack: CUP_B_Kombat_Radio_Olive
+    {
+        scope=1;
+        class TransportMagazines
+        {
+            class _xx_1Rnd_HE_Grenade_shell {count=11;magazine="1Rnd_HE_Grenade_shell";};
```

Comment:
There are macros to do this more nicely that you can use. https://github.com/uksf/modpack/blob/0d65b82f0568a3ea6d465f3e5fb2f760af1452b0/addons/main/script_macros.hpp#L69
e.g https://github.com/uksf/air/blob/81b1d4699d6d1520eff952ee7142fecb78a5e2e1/addons/patches/helis/CfgMerlin.hpp#L118